IFC4x3/Sections/Domain specific data schemas/Schemas/IfcElectricalDomain/PropertySets/Pset_ElectricalDeviceCommon/DocPropertySet.xml

```diff
@@ -14,6 +14,11 @@
 		<DocProperty xsi:nil="true" href="InsulationStandardClass_1TFB_0qTyHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="IP_Code_1MhaW0qTyHuO00025QrEV" />
 		<DocProperty xsi:nil="true" href="IK_Code_2kpG9_rK93LwNz4pCQcRnH" />
+		<DocProperty xsi:nil="true" href="EarthingStyle_31GzGB0Kz6DfNA40OdlMGX" />
+		<DocProperty xsi:nil="true" href="HeatDissipation_0lf5nUXkv7RfqST9JcFatz" />
+		<DocProperty xsi:nil="true" href="MaximumPower_3GU6eTlpT678UXrRfzKcYW" />
```

Comment:
would it be worth calling this RatedPower to match the first 2 properties which are all interrelated

or would it be NominalPowerConsumption changed to RatedPower & MaximumPower is a higher burst load rating??

Reply:
resolved during meeting